.trunk/trunk.yaml

```diff
@@ -18,7 +18,7 @@ plugins:
 runtimes:
     enabled:
         - go@1.21.0
-        - node@18.18.0
+        - node@20.19.0
```

Comment:
This looks like it was intentionally set to the min supported version, so I did the same.